openvdb_points/tools/PointAttribute.h

```diff
@@ -59,13 +60,15 @@ namespace tools {
 /// @param hidden        mark attribute as hidden
 /// @param transient     mark attribute as transient
 /// @param group         mark attribute as group
+/// @param string        mark attribute as string
 template <typename PointDataTree>
 inline void appendAttribute(PointDataTree& tree,
                             const AttributeSet::Util::NameAndType& newAttribute,
                             Metadata::Ptr defaultValue = Metadata::Ptr(),
                             const bool hidden = false,
                             const bool transient = false,
-                            const bool group = false);
+                            const bool group = false,
+                            const bool string = false);
```

Comment:
Not a huge fan of this function signature, can restructure these so they are templated on the appended type? I mentioned this to you before, have you have any thoughts? Also what was the reasoning again for having functions like these templated on the tree type?